scripts/issue/502/top_ups_for_q3.py

```diff
@@ -0,0 +1,86 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    safe = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    trops = GreatApeSafe(r.badger_wallets.treasury_ops_multisig)
+    badger = safe.contract(r.treasury_tokens.BADGER)
+    usdc = safe.contract(r.treasury_tokens.USDC)
+    weth = safe.contract(r.treasury_tokens.WETH)
+    wbtc = safe.contract(r.treasury_tokens.WBTC)
+    bal = safe.contract(r.treasury_tokens.BAL)
+    usdt = safe.contract(r.treasury_tokens.USDT)
+    dai = safe.contract(r.treasury_tokens.DAI)
+    eurs = safe.contract(r.treasury_tokens.EURS)
+    crv = safe.contract(r.treasury_tokens.CRV)
+    cvx = safe.contract(r.treasury_tokens.CVX)
+    fei = safe.contract(r.treasury_tokens.FEI)
+    angle = safe.contract(r.treasury_tokens.ANGLE)
+    threepool = safe.contract(r.treasury_tokens.crv3pool)
+    ibbtc_lp = safe.contract(r.treasury_tokens.crvIbBTC)
+    slp = safe.contract(r.sett_vaults.bslpWbtcibBTC)
+    bpt_badgerwbtc = safe.contract(r.balancer.B_20_BTC_80_BADGER)
+    bpt_3pool = safe.contract(r.balancer.B_3POOL)
+    bvecvx = safe.contract(r.treasury_tokens.bveCVX)
+
+    safe.take_snapshot([
+        badger, usdc, weth, bal, usdt, dai, eurs, crv, cvx, fei, angle,
+        threepool, ibbtc_lp, slp, bpt_badgerwbtc, bpt_3pool, bvecvx
+    ])
+    trops.take_snapshot([
+        badger, usdc, weth, bal, usdt, dai, eurs, crv, cvx, fei, angle,
+        threepool, ibbtc_lp, slp, bpt_badgerwbtc, bpt_3pool, bvecvx
+    ])
+
+    # badger for emissions, rembadger, bribes and contribs q3
+    badger.transfer(r.drippers.tree_2022_q2, 260_000e18)
```

Comment:
maybe for the drippers already use dummy placers as `r.drippers.tree_2022_q3` & `r.drippers.rembadger_2022_q3` even if using same address as q2, so if it gets merged before deploying the q3 ones, then it just matter of updating the address book or worth deploying them already and include them in this pr

Reply:
pushed in eb1249f